services/app-web/src/testHelpers/s3Helpers.ts

```diff
@@ -17,7 +17,7 @@ export const testS3Client: () => S3ClientT = () => {
             return key instanceof Error ? null : key
         },
         getS3URL: async (s3key: string, fileName: string): Promise<string> => {
-            return `fakes3://fake-bucket/${s3key}/${fileName}`
+            return `s3://fake-bucket/${s3key}/${fileName}`
```

Comment:
Had to change this due to test failures for improperly formatted keys. We have a function `isValidS3URLFormat` that was throwing an error from the mocked key that didn't follow the expected format and then broke a test